src/components/Header.spec.js

```diff
@@ -0,0 +1,10 @@
+import {render, screen} from '@testing-library/react'
+  import Header from './Header'
+    
+  describe('Header', () => {
+      it('renders', () => {
+          render(<Header />)
+          expect(screen.getByText('Header')).toBeInTheDocument()
```

Comment:
Are you sure that this test passes? `Header` is not rendering the text `Header` or did I miss something?

Reply:
Correct test